### ISSUE TITLE
Tab is not shown/displayed properly when selected

### ISSUE THREAD

**Issue by roger7467** [a85/POSTMan-Chrome-Extension#5840]
Please read through the [guidelines](https://github.com/postmanlabs/postman-app-support#guidelines-for-reporting-issues) before creating a new issue.

**Describe the bug**
When I click on a request in the collection, the request details are not displayed on the right-hand side.

**To Reproduce**
1. Go to any request in the collection
2. Click on any request
3. Nothing is shown on the right-hand side

**Expected behavior**
Show request details

**Screenshots**
![image](https://user-images.githubusercontent.com/695876/51839837-ad8f3700-230a-11e9-8a9d-cec87d232ce3.png)

**App information (please complete the following information):**
 - App Type [Native App]
 - Postman Version [6.7.2]
 - OS: [Win10]

**Additional context**

**Comment by roger7467** [comment]
This is quite urgent and blocking my test.
Anyone there?

**Comment by sujayvenaik** [comment]
Hey @roger7467 

Could you send us the logs from View -> Developer -> Show DevTools (Current View)?

Also, for quickly unblocking yourself you can open requests in a new tab. (Right-click on request name and press Open in New tab)

**Comment by roger7467** [comment]
![image](https://user-images.githubusercontent.com/695876/51851271-d7088c80-2323-11e9-83f3-465599fc74ce.png)

**Comment by roger7467** [comment]
[-1548693763982.log](https://github.com/postmanlabs/postman-app-support/files/2803972/-1548693763982.log)

**Comment by sujayvenaik** [comment]
@roger7467  Thanks for the logs.
We will debug this further and update this thread.

**Comment by numaanashraf** [comment]
Fixed in Postman v7.0
